README.rst

```diff
@@ -20,12 +20,12 @@ zyte-common-items
 
 .. description starts
 
-``zyte-common-items`` is a Python 3.7+ library of item classes used by Zyte_ to
-normalize different types of data extracted from websites.
-
-It can be used in custom data extraction code for normalization purposes,
-maximizing opportunities for code reuse.
+``zyte-common-items`` is a Python 3.7+ library of item_ and `page object`_
```

Comment:
a side note: we might need to think about a better name for this package

Reply:
zyte-poet?

Reply:
What about `zyte-commons`? or `zyte-reuse`? `zyte-building-blocks`?

Something that could possibly accommodate other things in the future that we haven't thought about.

Reply:
Like those first 2.

Playing with `zyte-reuse`, I did thought of `rezyte`, which would be very much in line with anything “poet”, but I still like your suggestions much better :smile: